tox.ini

```diff
@@ -1,24 +1,42 @@
 [tox]
 args_are_paths = false
 skip_missing_interpreters = true
-envlist = py{37,38,39,py3}-dj22, \
-          py{37,38,39,py3}-dj32, \
-          flake8
+envlist = 
+    py3.9-flake8
+    {py3.7,py3.8,py3.9}-django22
+    {py3.7,py3.8,py3.9,py3.10}-django32
+    {py3.8,py3.9,py3.10}-django40
+    ; DISABLED: needs a viable way to upgrade pypy-bundled sqlite to be 3.8.3, 
+    ; not 3.7.17, else Django won't start
+    ; {pypy3.7,pypy-3.8}-django22
+    ; {pypy-3.7,pypy-3.8}-django32
+    ; {pypy-3.8}-django40
 
 [testenv]
 usedevelop = true
 pip_pre = true
 commands = ./runtests.py
 deps =
-    dj22: Django>=2.2,<2.3
-    dj32: Django>=3.2,<4.0
+    django22: Django>=2.2,<2.3
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<5.0
```

Comment:
4.1 is slated for Aug 2022 so I'd do `Django>=4.0,<4.1` to avoid Django 4.1 here.